packages/genesys-spark-components/src/components/stable/gux-form-field/components/gux-form-field-time-picker/gux-form-field-time-picker.tsx

```diff
@@ -105,21 +106,26 @@ export class GuxFormFieldTimePicker {
   render(): JSX.Element {
     return (
       <GuxFormFieldFieldsetContainer labelPosition={this.computedLabelPosition}>
-        <GuxFormFieldLegendLabel
-          position={this.computedLabelPosition}
-          required={this.required}
-          labelText={this.label?.textContent}
+        <GuxFormFieldLegendLabelContainer
+          labelPosition={this.computedLabelPosition}
         >
-          <slot name="label" onSlotchange={() => this.setLabel()} />
-          {this.renderScreenReaderText(
-            this.getI18nValue('required'),
-            this.required
-          )}
-          {this.renderScreenReaderText(
-            getSlotTextContent(this.root, 'error'),
-            this.hasError
-          )}
-        </GuxFormFieldLegendLabel>
+          <GuxFormFieldLegendLabel
+            required={this.required}
+            labelText={this.label?.textContent}
```

Comment:
Do we want default label text incase there is no label defined?

Reply:
I don't think we want a label default here.